### DISCUSSION TITLE
How to remove the navigation from all pages?

### DISCUSSION THREAD

**Discussion by glow12121** [datenstrom/community#836] [Ask a question] [answered]
I am making a small page and would like to delete the upper navigation bar from all pages. What is the best way to do it? The theme is the modified Berlin.

And an adjacent question please. Alternatively, how do I keep the name of the website in the top bar - but remove all other titles?

**Comment by schulle4u** [comment, marked as answer]
You can remove the top navigation by opening your `system/layouts/header.html`. Simply remove the corresponding line for the navigation: 

`<?php $this->yellow->layout("navigation") ?>`

See also: [How to customise a layout](https://datenstrom.se/yellow/help/how-to-customise-a-layout)

**Reply by glow12121** [reply]
Thank you very much, this has worked.
